### PR TITLE
revamp of readme.

### PR DESCRIPTION
**Pull request by sinzone**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,8 +43,12 @@ freenode: [#kong](http://webchat.freenode.net/?channels=kong)
   - **ACL**: Acccess Control for your API Consumers.
   - **IP-restriction**: Whitelist or blacklist IPs that can make requests.
   - **Response-Rate-Limiting**: Rate limiting based on custom response header value.
-  - **Analytics**: Visualize, Inspect and Monitor API traffic with [Galileo](https://getgalileo.io).
-  - **SSL**: Setup a specific SSL certificate for an underlying service or API.
+  - **API Analytics**: Visualize, Inspect and Monitor API traffic with [Galileo](https://getgalileo.io).
+  - **Dev Portal**: [Gelato](https://getgelato.io) builds beutiful Dev Portal and on-board developers easily.
```
Comment (akurkin) [inline README.md]: typo here, "beautiful"

Comment (sinzone) [inline README.md]: thx. fixed.

File: README.md [hunk 2]
```diff
@@ -105,12 +118,19 @@ Tools:
 - [Konga CLI Tool](https://github.com/Floby/konga-cli)
 - [Kong on Tutum](https://github.com/Sillelien/docker-kong)
 - [Kong GUI in JS](https://github.com/rsdevigo/jungle)
-- [Kong GUI in Py](https://github.com/vikingco/django-kong-admin)
+- [Kong Admin in Python](https://github.com/vikingco/django-kong-admin)
 - [Kong UI](https://github.com/msaraf/kong-ui)
 - [Chef Cookbook for Kong](https://github.com/zuazo/kong-cookbook)
+- [Puppet recipie for Kong](https://github.com/scottefein/puppet-nyt-kong)
```
Comment (thibaultCha) [inline README.md]: typo: recipe

File: README.md [hunk 3]
```diff
@@ -105,12 +118,19 @@ Tools:
 - [Konga CLI Tool](https://github.com/Floby/konga-cli)
 - [Kong on Tutum](https://github.com/Sillelien/docker-kong)
 - [Kong GUI in JS](https://github.com/rsdevigo/jungle)
-- [Kong GUI in Py](https://github.com/vikingco/django-kong-admin)
+- [Kong Admin in Python](https://github.com/vikingco/django-kong-admin)
```
Comment (thibaultCha) [inline README.md]: Kong Admin but others say Kong UI or Kong GUI. They should probably all say "Kong Admin UI"

File: README.md [hunk 4]
```diff
@@ -105,12 +118,19 @@ Tools:
...
 - [Kong UI](https://github.com/msaraf/kong-ui)
 - [Chef Cookbook for Kong](https://github.com/zuazo/kong-cookbook)
+- [Puppet recipie for Kong](https://github.com/scottefein/puppet-nyt-kong)
+- [Ansible role for Kong/Ubuntu](https://github.com/Getsidecar/ansible-role-kong)
 - [Python Client for Kong](https://pypi.python.org/pypi/python-kong/)
 - [Kong with Instaclustr](https://www.instaclustr.com/products/kong/)
 - [.NET Client for Kong](https://www.nuget.org/packages/Kong/0.0.4)
+- [Bonobo: Kong Key Managment (with Mashery migration scripts)](https://github.com/guardian/bonobo)
+- [Adds Monit to base Kong Image](https://github.com/articulate/docker-kong-wait)
```
Comment (thibaultCha) [inline README.md]: "Kong image with monitoring support for Cassandra"?

File: README.md [hunk 5]
```diff
@@ -105,12 +118,19 @@ Tools:
...
+- [Kong for CanopyCloud](https://github.com/CanopyCloud/cip-kong)
+- [Extended File Loggin Plugin](https://github.com/nazarhussain/kong-file-log-extended)
```
Comment (thibaultCha) [inline README.md]: typo: File logging?

File: README.md (hunk 4 above)
Comment (sinzone) [inline README.md]: suggesting to change the monit title?

Comment (thibaultCha) [inline README.md]: suggesting to be more descriptive as to what the image actually do, aka what it is monitoring for.

"Kong image waiting for Cassandra " works too, is less abstract.

**Comment by sinzone** [comment]
all good

**Comment by thibaultCha** [comment]
Manually applied on master + next.

**Pull request closed without merging**